packages/common/src/store/ui/own-profile/slice.ts

```diff
@@ -0,0 +1,30 @@
+import type { PayloadAction } from '@reduxjs/toolkit'
+import { createSlice } from '@reduxjs/toolkit'
+
+import { Nullable } from 'utils/index'
+
+export type OwnProfileState = typeof initialState
+
+type State = {
+  trackCount: Nullable<number>
+}
+
+const initialState: State = {
+  trackCount: null
+}
+
+const slice = createSlice({
+  name: 'ownProfile',
```

Comment:
wondering if this should be part of account?
perhaps `store.account.hasTracks`

Reply:
yeah i could def see this being a part of account actually!